README.md

```diff
@@ -246,6 +246,21 @@ The command below can be used to run the glibc tests:
 
     make check-glibc-linux
 
+##### Adding more arch/abi combination for testing without introducing multilib
+
+`--with-extra-multilib-test` can be used when you want to test more combination
+of arch/ABI, for example: built a linux toolchain with multilib with
+`rv64gc/lp64d` and `rv64imac/lp64`, but you want to test more configuration like
+`rv64gcv/lp64d` or `rv64gcv_zba/lp64d`, then you can use --with-extra-multilib-test
+to specify that via `--with-extra-multilib-test="rv64gcv-lp64d;rv64gcv_zba-lp64d"`,
+then the testing will run for `rv64gc/lp64d`, `rv64imac/lp64`, `rv64gcv/lp64d`
+and `rv64gcv_zba/lp64d`.
```

Comment:
IIUC, you need to build a superset of the configuration. The example in the README would need to be swapped/modified - for example:
Build multilib toolchain with:
`rv64gcv_zba/lp64d` and `rv32iac-ilp32`
Test with (subset of build-multilibs):
`rv64gc/lp64d` or `rv32i/ilp32`

 `--with-extra-multilib-test="rv64gcv-lp64d;rv32i-ilp32"`
 
 IIUC, this will combine the build multilib target with the `extra-multilib` tests and test these configurations:
 `rv64gcv_zba-lp64d;rv64gc-lp64d;rv32iac-ilp32;rv32i-ilp32`
 
Is that correct?

Reply:
Actually subset rather than superset, otherwise you still need target has capability with superset, e.g. test `rv64gc` but library built with `rv64gcv_zba`, so you still need `rv64gcv_zba` for target